docs/analyzer.md

```diff
@@ -0,0 +1,25 @@
+# Analyzer
+
+FakeItEasy provides a Roslyn analyzer to detect incorrect usages of the library,
+which cannot be prevented by the API or the compiler but will typically result in
+bugs at runtime.

```

Comment:
", which" sounds to me like no incorrect usages of the library can be prevented by the API or the compiler.
I think we want a [restrictive clause](http://www.quickanddirtytips.com/education/grammar/which-versus-that-0) formed by "that", indicating the the analyzer is intended to help with the subset of incorrect usages that can't be prevented by the API or compiler.

Unless I've misinterpreted your intent?